tools/install.py

```diff
@@ -51,21 +51,45 @@ def run(target, local, user, config, toolchain, **kwargs):
 
     # Target location to install to
     if local:
+        target_name = target
```

Comment:
this is almost certainly beyond the scope of this PR, but I think `target` is kind of confusing here, since that's a word we use for other things. It's `./b target generic`, then in the cmake scripts, a target is a thing we build, then here it's the location we're installing to. Idk. It's a bit to juggle